zap/src/test/resources/org/zaproxy/zap/spider/parser/htmlform/FormWithFormactionButtons.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Form With Button Using Formaction - Spider HTML Form Parser</title>
+</head>
+<body>
+
+<form action="http://example.org/form1" method="%%METHOD%%">
+    <input type="text" name="field1" value="field1" />
+    <input type="text" name="field2" value="field2" />
+    <button formaction="http://example.org/formaction1"></button>
+</form>
+
+<form action="http://example.org/form2" method="%%METHOD%%">
+    <input type="text" name="field1" value="field1" />
+    <input type="text" name="field2" value="field2" />
+    <button>Default Action</button>
+    <button formaction="http://example.org/formaction2">FormAction2</button>
+</form>
+
+<form action="http://example.org/form3" method="%%METHOD%%">
```

Comment:
It was a test to verify that the correct formaction was picked up in case of no buttons, but since this test case is already covered by resources/org/zaproxy/zap/spider/parser/htmlform/FormNoSubmitField.html, there is no need for a duplicate indeed. I removed it